src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -387,6 +387,25 @@ source %{
     return false;
   }
 
+  // Return true if vector mask operation with "opcode" requires the mask to be
+  // saved in a predicate register.
+  bool Matcher::vector_mask_requires_predicate(int opcode, const TypeVect* vt) {
```

Comment:
What would be the alternative, if it is not in a predicate register?

Reply:
It will be in a vector register like architectures that do not have predicate feature such as Arm NEON and X86 AVX1/2.